lib-ee/emqx_license/src/emqx_license_resources.erl

```diff
@@ -128,6 +128,6 @@ ensure_timer(#{check_peer_interval := CheckInterval} = State) ->
 
 remote_connection_count() ->
     Nodes = mria_mnesia:running_nodes() -- [node()],
-    Results = emqx_license_proto_v1:remote_connection_counts(Nodes),
+    Results = emqx_license_proto_v2:remote_connection_counts(Nodes),
```

Comment:
Is something needs to be done for backward compatibility with v1?

Reply:
No, this API didn't change.  I thought we needed to bump all uses of `v1` when a new proto version came out.  :hear_no_evil: 

But now that you mention it, we probably don't need to?

Reply:
As we talked, it's fine to bump this version.